framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/EntityFrameworkCore/DependencyInjection/AbpDbContextConventionalRegistrar.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Reflection;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Volo.Abp.DependencyInjection;
+
+namespace Volo.Abp.EntityFrameworkCore.DependencyInjection
+{
+    public class AbpDbContextConventionalRegistrar : DefaultConventionalRegistrar
```

Comment:
We should not create a registrar. We can do it much simpler.

See this:

https://github.com/abpframework/abp/blob/8770fd7811de54b40762de3df11b84d8c7f8c443/framework/src/Volo.Abp.EntityFrameworkCore/Microsoft/Extensions/DependencyInjection/AbpEfCoreServiceCollectionExtensions.cs#L10-L29

Here, before `optionsBuilder?.Invoke(options);`, we can get the ReplaceDbContextAttribute attribute (or attributes, because there is no restriction - developer may use it multiple times instead of passing multiple types into a single attribute), then call `ReplaceDbContext` for each replaced context. Same for mongdb too.